tests/testthat/test_varfield.R

```diff
@@ -166,3 +168,250 @@ test_that('psdest produces equivalent results to manual calculation.',
     psd2 <- psdest(list(reof, reof))
     expect_equal(sqrt(psd2$psd), Fxmag)
 })
+
+
+###############################
```

Comment:
Sometime when we have time we should come up with some downsized test data sets.  Right now our tests take a really long time, but I think we could test the functionality just as well and much more quickly with some smaller datasets.